### PR TITLE
[#9480] Add instructor course details page snapshot tests

### PR DESCRIPTION
**Pull request by monmanuela**
Part of #9480 

**Outline of Solution**
Test cases added:
- default
- a course with instructor and no student
- a course with instructor and one student

### CODE REVIEW

File: src/web/app/pages-instructor/instructor-course-details-page/__snapshots__/instructor-course-details-page.component.spec.ts.snap [hunk 1]
```diff
@@ -0,0 +1,422 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InstructorCourseDetailsPageComponent should snap with a course with one co-owner and no students 1`] = `
+<tm-instructor-course-details-page
+  clipboardService={[Function ClipboardService]}
+  courseDetails={[Function Object]}
+  courseService={[Function CourseService]}
+  courseStudentListAsCsv=""
```
Comment (AyushChatto) [inline src/web/app/pages-instructor/instructor-course-details-page/__snapshots__/instructor-course-details-page.component.spec.ts.snap]: Might want to include a case with this populated so we know it is rendered correctly

**Comment by monmanuela** [comment]
@AyushChatto Done 👍

**Comment by damithc** [comment]
> I don't see the `'Failed to load. Please reopen modal.` message anywhere that should ideally be rendered? Is this a bug?

On a related note, `modal` is a technical term; should not be used in a user-facing message.

**Comment by monmanuela** [comment]
> I don't see the `'Failed to load. Please reopen modal.` message anywhere that should ideally be rendered? Is this a bug?
> 
> And now that I look at the HTML page again, maybe we should have a case where `loading` is true too.

Hmm, I don't think so. That message is in `<ng-template #studentTableModal>` and it's only triggered when the `Click Here` button is clicked. Since the snapshot test doesn't click the button, the modal wouldn't appear. Same goes for `loading`.

**Pull request merged**